tests/components/abode/test_cover.py

```diff
@@ -0,0 +1,65 @@
+"""Tests for the Abode cover device."""
+from unittest.mock import patch
+
+from homeassistant.components.abode import ATTR_DEVICE_ID
+from homeassistant.components.cover import DOMAIN as COVER_DOMAIN
+from homeassistant.const import (
+    ATTR_ENTITY_ID,
+    ATTR_FRIENDLY_NAME,
+    SERVICE_CLOSE_COVER,
+    SERVICE_OPEN_COVER,
+    STATE_CLOSED,
+)
+
+from .common import setup_platform
+
+DEVICE_ID = "cover.garage_door"
+
+
+async def test_entity_registry(hass):
+    """Tests that the devices are registered in the entity registry."""
+    await setup_platform(hass, COVER_DOMAIN)
+    entity_registry = await hass.helpers.entity_registry.async_get_registry()
+
+    entry = entity_registry.async_get(DEVICE_ID)
+    assert entry.unique_id == "61cbz3b542d2o33ed2fz02721bda3324"
+
+
+async def test_attributes(hass):
+    """Test the cover attributes are correct."""
+    await setup_platform(hass, COVER_DOMAIN)
+
+    state = hass.states.get(DEVICE_ID)
+    assert state.state == STATE_CLOSED
+    assert state.attributes.get(ATTR_DEVICE_ID) == "ZW:00000007"
+    assert not state.attributes.get("battery_low")
+    assert not state.attributes.get("no_response")
+    assert state.attributes.get("device_type") == "Secure Barrier"
+    assert state.attributes.get(ATTR_FRIENDLY_NAME) == "Garage Door"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5184102)